docs/plugins/api/bdapi.md

```diff
@@ -118,37 +118,37 @@ id|string|&#x274C;|*none*|Setting ID in the category
 ___
 
 ### findAllModules <span class="deprecated">Deprecated</span>
-Finds multple webpack modules using a filter
+Finds multiple webpack modules using a filter.
 
 | Parameter |  Type  |       Description      |
 |:----------|:------:|:----------------------:|
-filter|function|A filter given the exports, module, and moduleId. Returns true if the module matches.
+filter|function|A filter given the exports, module, and moduleId. Returns `true` if the module matches.
 
 **Returns:** `Array` - Either an array of matching modules or an empty array
 ___
 
 ### findModule <span class="deprecated">Deprecated</span>
-Finds a webpack module using a filter
+Finds a webpack module using a filter.
 
 | Parameter |  Type  |       Description      |
 |:----------|:------:|:----------------------:|
-filter|function|A filter given the exports, module, and moduleId. Returns true if the module matches.
+filter|function|A filter given the exports, module, and moduleId. Returns `true` if the module matches.
 
 **Returns:** `any` - Either the matching module or `undefined`
 ___
 
 ### findModuleByDisplayName <span class="deprecated">Deprecated</span>
-Finds a webpack module by displayName property
+Finds a webpack module by its `displayName` property.
```

Comment:
```suggestion
Finds a webpack module by it's `displayName` property.
```

Reply:
The "its" in this case is a possessive determiner (like his, hers etc.), not the words "it" and "is".

Reply:
correct, so it still needs the apostrophe to my knowledge